kythe/go/extractors/govname/types.go

```diff
@@ -24,44 +24,44 @@ import (
 )
 
 // BasicType returns the VName for a basic builtin Go type.
-func BasicType(b *types.Basic) *spb.VName {
-	return &spb.VName{Corpus: golangCorpus, Language: Language, Signature: b.Name() + "#builtin"}
+func BasicType(corpus string, b *types.Basic) *spb.VName {
```

Comment:
@schroederc do you think it's worth making an `Options` struct to be a receiver for these functions? Might be slightly cleaner than passing the corpus as an argument, but I'll need to make sure the `Options` struct is then available at each place these functions are called. I don't foresee the need for other options besides corpus for now.

Reply:
Yeah, that sounds reasonable.  Maybe `Config`?